pyproject.toml

```
@@ -31,6 +31,7 @@ isort = "^5.9.3"
 pre-commit = "^2.14.1"
 pyupgrade = "^2.25.0"
 seed-isort-config = "^2.2.0"
+tox = "^3.24.3"
 
 [build-system]
 requires = ["poetry-core>=1.0.0"]
```

Github doesn't allow me to add a comment there, but if the oldest version is 3.7 Black target version should be updated
```suggestion
[tool.black]
line-length = 120
target-version = ['py37']
```